library/extractor/src/main/java/com/google/android/exoplayer2/extractor/ogg/OggPacket.java

```diff
@@ -29,7 +29,11 @@
 /** OGG packet class. */
 /* package */ final class OggPacket {
 
-  private final OggPageHeader pageHeader = new OggPageHeader();
+  private final OggPageHeader pageHeader;
+  public OggPacket(long streamSerialNumber) {
+    pageHeader = new OggPageHeader(streamSerialNumber);
+  }
```

Comment:
Javadoc is missing. Also this constructor should be moved after all variables are declared.